fetch.bs

```diff
@@ -5536,6 +5539,9 @@ optional boolean <var>forceNewConnection</var> (default false), run these steps:
 
        <li><p>Let <var>status</var> be the HTTP response's status code.
```

Comment:
@annevk - Would it make sense to also explicitly set the response's status to |status| here? (maybe as a separate PR)

Reply:
It might be worth having a follow-up issue on how to better integrate with HTTP or indicate that's what's happening. We basically want to map an HTTP response to a Fetch response. There has been some discussion about this also with the HTTP WG, but I don't think we'll get anything formal out of that.